presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergAbstractMetadata.java

```diff
@@ -546,6 +560,18 @@ protected List<ColumnMetadata> getColumnMetadatas(Table table)
                 .collect(toImmutableList());
     }
 
+    protected List<ColumnMetadata> getColumnMetadatas(View view)
```

Comment:
```suggestion
    protected List<ColumnMetadata> getColumnMetadata(View view)
```

Small nit, metadata is already plural.